app/src/main/java/de/westnordost/streetcomplete/quests/building_type/BuildingTypes.kt

```diff
@@ -58,6 +58,10 @@ enum class BuildingType(val item: Item<String>) {
     SERVICE       (Item("service",    R.drawable.ic_building_service,    R.string.quest_buildingType_service, R.string.quest_buildingType_service_description)),
     HANGAR        (Item("hangar",     R.drawable.ic_building_hangar,     R.string.quest_buildingType_hangar, R.string.quest_buildingType_hangar_description)),
     BUNKER        (Item("bunker",     R.drawable.ic_building_bunker,     R.string.quest_buildingType_bunker)),
+    //  TODO: replace placeholder icons
```

Comment:
I'll do that after the merge

Reply:
Should I remove this comment before updating the PR?